dev-packages/browser-integration-tests/suites/tracing/trace-lifetime/pageload/test.ts

```diff
@@ -48,3 +52,25 @@ sentryTest('error after pageload has pageload traceId', async ({ getLocalTestPat
   const errorTraceId = errorEvent.contexts?.trace?.trace_id;
   expect(errorTraceId).toBe(pageloadTraceId);
 });
+
+sentryTest('error during pageload has pageload traceId', async ({ getLocalTestPath, page }) => {
+  if (shouldSkipTracingTest()) {
+    sentryTest.skip();
+  }
+
+  const url = await getLocalTestPath({ testDir: __dirname });
+
+  const envelopeRequestsPromise = getMultipleSentryEnvelopeRequests<Event>(page, 2);
+  const [, , events] = await Promise.all([page.goto(url), page.locator('#errorBtn').click(), envelopeRequestsPromise]);
```

Comment:
I still think this pattern will cause more race conditions than prevent them.
```suggestion
  await page.goto(url);
  await page.locator('#errorBtn').click();
  const events = await envelopeRequestPromise;
```

Reply:
The important thing is that `page.goto` and `page.locator.click` happen as fast as possible after each other. I guess it doesn't make much difference though if this happens in `Promise.all` vs sequential awaits. 
Awaiting the promise individually afterwards definitely works - will update the two tests.